cmd/meroxa/root/resources/describe.go

```diff
@@ -66,7 +66,13 @@ func (d *Describe) Execute(ctx context.Context) error {
 	d.logger.Info(ctx, utils.ResourceTable(resource))
 
 	if tun := resource.SSHTunnel; tun != nil {
-		d.logger.Info(ctx, "\nPaste the following public key on your host:")
+		nextSteps := "\nPaste the following public key on your host:"
+		if resource.Status.State == "error" {
+			d.logger.Info(ctx, "\nDeadline hit, no longer retrying to reconnect to resource.")
```

Comment:
Nit, I would assign this line to `nextSteps` too so that it's printed at once. This would make it cleaner as well:

```
nextSteps = "\nDeadline hit, no longer retrying to reconnect to resource.\nPlease initiate validation manually after pasting the following public key on your host:"
```